code/xcvm/evm/.gitignore

```diff
@@ -0,0 +1,14 @@
+# Compiler files
+cache/
+out/
+
+# Ignores development broadcast logs
+!/broadcast
+/broadcast/*/31337/
+/broadcast/**/dry-run/
+
+# Docs
+docs/
```

Comment:
Why are we ignoring docs?

Reply:
it is default generated by foundry. whole ignore. i do not know why they do it. will not dig into their reasons why. may be because of how they handle deps.

Reply:
i agree thay may be they overshoot, but will  not touch any default until they are blocker for now.